debian/cuttlefish-common.cuttlefish-host-orchestrator.init

```diff
@@ -34,19 +34,34 @@
 if [ -f /etc/default/cuttlefish-common ]; then
     . /etc/default/cuttlefish-common
 fi
-orchestrator_port=${orchestrator_port:-1080}
+orchestrator_http_port=${orchestrator_http_port:-1080}
+orchestrator_https_port=${orchestrator_https_port:-1443}
 
 RUN_DIR="/run/cuttlefish"
 ASSET_DIR="/usr/share/cuttlefish-common/host-orchestrator"
 DAEMON="/usr/lib/cuttlefish-common/bin/host-orchestrator"
+GENERATE_CERT="/usr/lib/cuttlefish-common/bin/generate_cert"
 PIDFILE="${RUN_DIR}"/host-orchestrator.pid
+CERT_DIR="/etc/cuttlefish-common/host-orchestrator/cert"
+
+function gen_cert {
+  mkdir -p "${CERT_DIR}"
+  cd "${CERT_DIR}"
+  # https://go.dev/src/crypto/tls/generate_cert.go
+  ${GENERATE_CERT} --host="localhost" --duration="867240h0m0s"
```

Comment:
this should check that the certificates don't exist before trying to create them, otherwise you'll create a new certificate on every boot.
Also, checking before creating will allow to provide your own certificate if you want to.

Reply:
Done